ports/nrf/common-hal/rotaryio/IncrementalEncoder.c

```diff
@@ -51,10 +51,10 @@ static void _intr_handler(nrfx_gpiote_pin_t pin, nrf_gpiote_polarity_t action) {
 
     // logic from the atmel-samd port: provides some damping and scales movement
     // down by 4:1.
-    if (self->quarter >= 4) {
+    if (self->quarter > 0 && new_state == 2) {
```

Comment:
Oh a finer point might be to refactor this to be ever so more efficient:
1. switch the order of the conditional `(new_state == 2 && self->quarter > 0)` because the `quarter` will be non-zero more often than the state is 2
2. I guess to go even further it could be refactored with an outer if clause and an inner case, perhaps like this: 
```
if (new_state == 2 && self->quarter != 0) {
    if (self->quarter > 0) { // increment
        ...
    } else if { // decrement 
        ...
    }
}
```

What do you think?